test/api/test_api.cpp

```diff
@@ -554,3 +554,18 @@ TEST_CASE("Issue #4583: Catch Insert/Update/Delete errors", "[api]") {
 	result = con.SendQuery("SELECT MIN(c0) FROM t0;");
 	REQUIRE(CHECK_COLUMN(result, 0, {1}));
 }
+
+TEST_CASE("Fuzzer issue #5984 no.25: C/C++ API Parsed statement fails", "[api]") {
+	DuckDB db(nullptr);
+	Connection con(db);
+	unique_ptr<QueryResult> result;
+
+	con.EnableQueryVerification();
+	REQUIRE_NO_FAIL(con.Query("CREATE TABLE t0 (c0 int);"));
```

Comment:
Can't we test this using the sqllogictests?

Reply:
Yep you're right, I've altered it!